Difi.SikkerDigitalPost.Klient.Testklient/Program.cs

```diff
@@ -156,7 +157,7 @@ private static Klientkonfigurasjon SettOppKlientkonfigurasjon()
         {
             var klientkonfigurasjon = new Klientkonfigurasjon();
             LeggTilLogging(klientkonfigurasjon);
-            klientkonfigurasjon.MeldingsformidlerUrl = new Uri("https://qaoffentlig.meldingsformidler.digipost.no/api/ebms");
+            klientkonfigurasjon.MeldingsformidlerUrl = new Uri("https://qaoffentlig.meldingsformidler.digipost.no/api/ebmHESTs");
```

Comment:
No reason. Bare lekte meg litt. Det er ikke så farlig, siden program bare er en lekegrind.

Mvh Aleksander Sjåfjell
-- Sendt fra min mobil




On Wed, Jun 17, 2015 at 3:10 AM -0700, "Kristian" <notifications@github.com<mailto:notifications@github.com>> wrote:


In Difi.SikkerDigitalPost.Klient.Testklient/Program.cs<https://github.com/difi/sikker-digital-post-klient-dotnet/pull/93#discussion_r32612062>:

> @@ -156,7 +157,7 @@ private static Klientkonfigurasjon SettOppKlientkonfigurasjon()
>          {
>              var klientkonfigurasjon = new Klientkonfigurasjon();
>              LeggTilLogging(klientkonfigurasjon);
> -            klientkonfigurasjon.MeldingsformidlerUrl = new Uri("https://qaoffentlig.meldingsformidler.digipost.no/api/ebms");
> +            klientkonfigurasjon.MeldingsformidlerUrl = new Uri("https://qaoffentlig.meldingsformidler.digipost.no/api/ebmHESTs");


hvorfor ?

—
Reply to this email directly or view it on GitHub<https://github.com/difi/sikker-digital-post-klient-dotnet/pull/93/files#r32612062>.